generate-docs/tools/API Coverage Report.csv

```diff
@@ -2094,34 +2094,34 @@ ExcelScript.PivotAxis,filter,EnumField,Poor,false
 ExcelScript.PivotAxis,row,EnumField,Poor,false
 ExcelScript.PivotAxis,unknown,EnumField,Poor,false
 ExcelScript.PivotDateFilter,N/A,Class,Good,false
-ExcelScript.PivotDateFilter,comparator,Method,Good,false
-ExcelScript.PivotDateFilter,condition,Method,Fine,false
-ExcelScript.PivotDateFilter,exclusive,Method,Good,false
-ExcelScript.PivotDateFilter,lowerBound,Method,Fine,false
-ExcelScript.PivotDateFilter,upperBound,Method,Fine,false
-ExcelScript.PivotDateFilter,wholeDays,Method,Fine,false
+ExcelScript.PivotDateFilter,comparator,Property,Good,false
+ExcelScript.PivotDateFilter,condition,Property,Fine,false
+ExcelScript.PivotDateFilter,exclusive,Property,Good,false
+ExcelScript.PivotDateFilter,lowerBound,Property,Fine,false
+ExcelScript.PivotDateFilter,upperBound,Property,Fine,false
+ExcelScript.PivotDateFilter,wholeDays,Property,Fine,false
 ExcelScript.PivotField,N/A,Class,Poor,false
-ExcelScript.PivotField,applyFilter(filter),Method,Good,true
-ExcelScript.PivotField,clearAllFilters(),Method,Good,false
-ExcelScript.PivotField,clearFilter(filterType),Method,Fine,false
+ExcelScript.PivotField,applyFilter(filter),Method,Fine,false
```

Comment:
Hmm, [this one](https://docs.microsoft.com/en-us/javascript/api/office-scripts/excelscript/excelscript.pivotfield?view=office-scripts#excelscript-excelscript-pivotfield-applyfilter-member(1)) does have an example but it's now being reported as false...

Reply:
Good catch. I'll investigate and backport to the other repo.

Reply:
Looks like in this repo, the property examples aren't tucked under remarks sections.